packages/shared/src/lib/query.ts

```diff
@@ -12,14 +12,15 @@ export type MutateFunc<T> = (variables: T) => Promise<(() => void) | undefined>;
 
 export const generateQueryKey = (
   name: string | RequestKey,
-  user: Pick<LoggedUser, 'id'> | null,
+  user: Pick<LoggedUser, 'id'> = null,
```

Comment:
Not blocking but do we need default null` here, since there is a check where no user always fallbacks to `anonymous`

Reply:
Seems unneeded indeed, moved back to old style.

Reply:
I mean, that's the point. Instead of doing `generateQueryKey(key, null)` we can just do `generateQueryKey(key)`.